### DISCUSSION TITLE
Allow getServerSideProps() for custom error (_error) pages

### DISCUSSION THREAD

**Discussion by ruhaim** [vercel/next.js#23345] [Ideas]
### Describe the feature you'd like to request

My Requirement
-My error page requires to be *translation enabled* AND dynamically generated (either via getServerSideProps() or getInitalProps()).

Few facts
-NextJS only allows getStaticProps() OR getInitialProps() for a custom _error page.
-next-i18next lib needs to be initialized via getServerSideProps() or getStaticProps() fns.

### Describe the solution you'd like

allow for getServerSideProps() for custom error (_error) pages

### Describe alternatives you've considered

I tried with getInitialProps() but I cannot get Translation to work since it (next-i18next) requires it to be initialized only in get(S)Sp functions

**Comment by shaohaolin** [comment]
Curious how you get the `getStaticProps()` working on custom error page.
